### PR TITLE
Made the android platform channel benchmarks comparable to iOS

### PR DESCRIPTION
**Pull request by gaaclarke**
Right now you can't compare iOS and Android tests because there is an extra copy on Android.  This makes them more directly comparable.

issue: https://github.com/flutter/flutter/issues/81559

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I signed the [CLA].
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test-exempt.
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

**Comment by gaaclarke** [comment]
@flutter-dashboard this is a test.

File: dev/benchmarks/platform_channels_benchmarks/android/app/src/main/kotlin/com/example/platform_channels_benchmarks/MainActivity.kt [hunk 1]
```diff
@@ -12,15 +12,21 @@ import io.flutter.plugin.common.StandardMessageCodec
 import java.nio.ByteBuffer
 
 class MainActivity: FlutterActivity() {
-
+    private var byteBufferCache : ByteBuffer? = null
```
Comment (dnfield) [inline dev/benchmarks/platform_channels_benchmarks/android/app/src/main/kotlin/com/example/platform_channels_benchmarks/MainActivity.kt:15]: nit: consider adding a comment about why we do this (e.g. what you have in your PR description).

**Review by dnfield — APPROVED** [review]
LGTM.

**Comment by Hixie** [comment]
test-exempt: is a test

**Pull request merged**